### PR TITLE
Release notes for 05-11 console release

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-05-11-console-release-notes-neondatabase.vercel.app/docs/release-notes

### CODE REVIEW

File: content/release-notes/2023-05-11-console.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Adjusted how Autoscaling minimum and maximum compute sizes are displayed in the **Compute endpoints** table on the branch details page.
+- UI: Added a **History retention** metric to the **Storage** widget on the Neon **Dashboard**. By default, Neon retains a 7-day history of data changes to support [point-in-time restore](../docs/reference/glossary#point-in-time-restore). History retention will become a configurable setting in a future release.
+- UI: Enhanced the **Request Downgrade** modal, which is accessible from the **Billing** page in the console.
```
Comment (PlayLikeNeverB4) [inline content/release-notes/2023-05-11-console.md:9]: Should we mention that it is now automated and the operation happens instantly?

File: content/docs/introduction/billing.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ The _Active time_ metric is a usage metric rather than a billing metric. It is a
...
+The _Compute time_ billing metric measures the amount of computing capacity used within a given time period. Neon periodically assesses compute utilization, at a minimum interval of 12 seconds, and calculates an average that reflects the observed computing capacity. Computing capacity is based on _Compute Units (CU)_. A CU in Neon is 1 vCPU and 4 GB of RAM. A Neon [compute endpoint](../reference/glossary/#compute-endpoint) can have anywhere from .25 to 7 CUs. A connection from a client or application activates a compute endpoint and its CUs. Activity on the connection keeps the compute endpoint and its CUs in an `Active` state. A defined period of inactivity places the compute endpoint and its CUs into an `Idle` state.
```
Comment (sharnoff) [inline content/docs/introduction/billing.md:98]: > at a minimum interval of 12 seconds

This phrasing feels awkward to me, but I don't have any better suggestions :(

**Review by sharnoff — DISMISSED** [review]

File: content/release-notes/2023-05-11-console.md (hunk 1 above)
Comment (danieltprice) [reply content/release-notes/2023-05-11-console.md]: Good idea. Please take a look at the revised note.

File: content/docs/introduction/billing.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/introduction/billing.md]: Thanks, Em. I have revised.

**Review by sharnoff — DISMISSED** [review]

**Pull request merged**